build/files.js

```diff
@@ -122,7 +117,7 @@ const headRegexp = /(^module.exports = \w+;?)/m
     ]
 
     , isBufferReplacement = [
-      /(\w+) instanceof Buffer/
+      /(\w+) instanceof Buffer/g
```

Comment:
because this is code that is coming in from io.js where they use instanceof so something to take up with them